### ISSUE TITLE
Using reCaptcha V2 and V3

### ISSUE THREAD

**Issue by sir-captainmorgan21** [DethAriel/ng-recaptcha#152]
### Summary
We are trying to use reCaptcha V3 and the visible reCaptcha V2. Everything was working with V3 no problem. When we added V2, we got an error (See error under description). If you or anyone has an example of both of them being used successfully, that would be great.

It looks like we should be able to do this. See "Can I run reCAPTCHA v2 and v3 on the same page?"
https://developers.google.com/recaptcha/docs/faq

I'm submitting a: 
- [x] bug report
- [x] feature request
- [ ] question / support request
- [ ] other

### Description
ERROR:
```
zone.js:199 Uncaught Error: reCAPTCHA has already been rendered in this element
    at Object.AE [as render] (VM3264 recaptcha__en.js:514)
    at RecaptchaComponent.push../node_modules/ng-recaptcha/recaptcha/recaptcha.component.js.RecaptchaComponent.renderRecaptcha (recaptcha.component.js:100)
    at SafeSubscriber._next (recaptcha.component.js:27)
    at SafeSubscriber.push../node_modules/rxjs/_esm5/internal/Subscriber.js.SafeSubscriber.__tryOrUnsub (Subscriber.js:194)
    at SafeSubscriber.push../node_modules/rxjs/_esm5/internal/Subscriber.js.SafeSubscriber.next (Subscriber.js:132)
    at Subscriber.push../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber._next (Subscriber.js:76)
    at Subscriber.push../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber.next (Subscriber.js:53)
    at BehaviorSubject.push../node_modules/rxjs/_esm5/internal/Subject.js.Subject.next (Subject.js:47)
    at BehaviorSubject.push../node_modules/rxjs/_esm5/internal/BehaviorSubject.js.BehaviorSubject.next (BehaviorSubject.js:38)
    at recaptcha-loader.service.js:49
```

PROPOSED SOLUTIONS:
It looks like we should be able to do this. See "Can I run reCAPTCHA v2 and v3 on the same page?"
https://developers.google.com/recaptcha/docs/faq

The problem is when you use the RecaptchaV3Module and implement it and then use the RecaptchaModule and use the Component, it results in a recaptcha script being loaded two times. According to their documentation, you should be able to load the script using your V3 site key, but still render the reCaptcha UI using the `render` function. So I think the RecaptchaLoaderService, should check to make sure any (not just V2) reCaptcha script hasn't already been loaded before attempting to load one itself.

Lib versions:

* ng-recaptcha: 4.3.0-beta.1
* Angular: 7.2.7
* Typescript (`tsc --version`): 3.2.4

**Comment by sir-captainmorgan21** [comment]
@DethAriel Do you mind looking into this? Would be very helpful. Or at least commenting on it?

**Comment by DethAriel** [comment]
hi @sir-captainmorgan21 !

Were you able to resolve the issue? If not, do you think the suggestion from [this comment](https://github.com/DethAriel/ng-recaptcha/issues/169#issuecomment-661845927) will help?

**Comment by rrayala-incomm** [comment]
Maybe something like cleanup() method can be implemented as said in the bellow issue:

https://github.com/google/recaptcha/issues/279

**Comment by DethAriel** [comment]
hello again @sir-captainmorgan21.

I'm glad to let you know that this bug has been fixed in `ng-recaptcha@6.0.2`, which is compatible with Angular versions 6 through 10.

In order to have proper v2 and v3 reCAPTCHA interoperability you should _always_ provide `RECAPTCHA_V3_SITE_KEY` (even in modules that only rely on v2 functionality). This will prevent bugs in your code by allowing `ng-recaptcha` to follow reCAPTCHA development guidelines properly ([this one](https://developers.google.com/recaptcha/docs/faq#can-i-run-recaptcha-v2-and-v3-on-the-same-page) in particular).

**Comment by sir-captainmorgan21** [comment]
@DethAriel Sorry. Forgot about this. But this is awesome. Thanks!
